pkg/sessionctx/stmtctx/stmtctx.go

```diff
@@ -732,6 +734,11 @@ func (sc *StatementContext) GetPlanHint() (string, bool) {
 	return sc.planHint, sc.planHintSet
 }
 
+// GetHasForce gets the HasForce boolean generated from the plan.
+func (sc *StatementContext) GetHasForce() bool {
```

Comment:
`Force` is too common and might bring confusion under the statement context, I suggest we can use a more concrete name like `IndexForce`.

Reply:
Done